src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -218,10 +218,10 @@ void AssertEmbeddedDSOs (string apk)
 		void AssertCompression (ZipEntry entry, bool compressed)
 		{
 			if (compressed) {
-				Assert.AreNotEqual (entry.CompressionMethod, CompressionMethod.Store, $"`{entry.FullName}` should be compressed!");
+				Assert.AreNotEqual (CompressionMethod.Store, entry.CompressionMethod, $"`{entry.FullName}` should be compressed!");
 				Assert.AreNotEqual (entry.Size, entry.CompressedSize, $"`{entry.FullName}` should be compressed!");
 			} else {
-				Assert.AreEqual (entry.CompressionMethod, CompressionMethod.Store, $"`{entry.FullName}` should be uncompressed!");
+				Assert.AreEqual (CompressionMethod.Store, entry.CompressionMethod, $"`{entry.FullName}` should be uncompressed!");
```

Comment:
FYI these parameters were the wrong way around. Expected should be first, actual second. Otherwise the error reporting it off.